src/main/java/Main.java

```diff
@@ -56,18 +51,41 @@ public static void main(String[] args) throws Exception {
                     System.out.println("your role");
                     String inputRole = scanner.nextLine();
                     UserRole createRole = UserRole.valueOf(inputRole.toUpperCase());
-                    new RegisterCommand().register(createUserName, createPassword, createName, createSurname, createRole);
-                    break;
-
-                case CommandConstants.EXIT_CODE:
+                    new RegisterCommand().
+                            execute(createUserName, createPassword, createName, createSurname, createRole);
+                }
+                case CommandConstants.EXIT_CODE -> {
                     ExitCommand exitCommand = new ExitCommand();
-                    exitCommand.exit();
-                    break;
-
-                case CommandConstants.SHOW_ME_LIST_CODE:
+                    exitCommand.execute();
+                }
+                case CommandConstants.SHOW_ME_LIST_CODE -> {
                     PrintUsersCommand printUsersCommand = new PrintUsersCommand();
                     printUsersCommand.print();
-                    break;
+                }
+                case CommandConstants.BAN_USER -> {
+                    if (UserSession.currentUser.getRole().equals(UserRole.ADMIN)) {
```

Comment:
1. Save currentRole to variable. And then use this variable in if
2. `enum.equals(value)` vs `value.equals(enum)` what is better?